packages/react-ui-ag/src/Modals/MoreOptionsModal.js

```diff
@@ -119,18 +119,16 @@ export default class MoreOptionsModal extends PureComponent {
           Reset All
         </Button>
         <div className={theme.MoreOptionsModal_FooterButtons}>
-          <Button
+          <CancelButton
             className={theme.MoreOptionsModal_Cancel}
             onClick={this.handleCancel}
-          >
-            Cancel
-          </Button>
-          <Button
+            name={'Cancel'}
```

Comment:
I really think we need to remove the name prop from the CancelButton and ApplyButton. It should just pass in children.

Reply:
Okie Dokie, I wasn't sure which one to do